templates/common/.github/workflows/bicep/azure-dev.yml

```diff
@@ -22,9 +22,9 @@ jobs:
     container:
       image: mcr.microsoft.com/azure-dev-cli-apps:latest
     env:
-      AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
-      AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
-      AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      AZURE_CLIENT_ID: ${{ vars.AZURE_CLIENT_ID }}
```

Comment:
Breaking change for external templates referencing secrets with new Azd bits? or do we handle gracefully? callout in changelog?

Reply:
We should call out the change in the changelog.  Folks using the older templates will need to update their templates to move to `${{ vars. }}` to `${{ secrets. }}`

Reply:
Yes, it would be a breaking change and users would need to move from `secrets.foo` to `vars.foo`.

We're not able to handle gracefully but we may be able to leverage the new `requiredVersions` param and set this to the RC version for all of our templates.